### PR TITLE
init: rust http server

### PR DESCRIPTION
**Pull request by PriyanshuPz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,47 @@
...
 
-1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).  
+1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/). Make sure that you have a `provider` installed.
 2. **Create the Workspace**:  
    ```bash  
-   daytona create <SAMPLE_REPO_URL> 
-   ```  
-
-... MORE STEPS IF NEEDED ...
+   daytona create https://github.com/priyanshuverma-dev/rust-http-server.git
```
Comment (idagelic) [inline README.md]: Update to daytonaio org repo instead of your personal one

**Review by idagelic — CHANGES_REQUESTED** [review]
Please also do a signoff on the commit
```
git commit --amend --signoff
git push -f
```

**Review by idagelic — APPROVED** [review]

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+			"extensions": [
+				"rust-lang.rust-analyzer",
+				"tamasfe.even-better-toml",
+				"serayuzgur.crates"
+			]
+		}
+	},
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "cargo run"
```
Comment (k0msenapati) [inline .devcontainer/devcontainer.json:20]: I guess it should be cargo build

**Comment by idagelic** [comment]
@kom-senapati Thanks for the suggestion - we will do tests and make sure everything works in the following steps.

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (PriyanshuPz) [reply .devcontainer/devcontainer.json:20]: Yes you are right but with `cargo build` it doesn't run the project only builds it. We can add that too. 👍

Comment (k0msenapati) [reply .devcontainer/devcontainer.json:20]: Yeah, in postCreateCommand we should add command for building only.

**Pull request merged**